src/App.css

```diff
@@ -1,39 +1,160 @@
-.App {
-  text-align: center;
+/* Navbar */
+nav {
+  background-color: #fff;
+  min-height: 6vh;
+  display: flex;
+  align-items: center;
+  font-size: calc(10px + 2vmin);
+  color: white;
+}
+
+nav h1 {
+  margin-left: 2%;
+  color: #0290ff;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+nav > div {
+  display: flex;
+  justify-content: space-between;
+  border: solid;
+  width: 20%;
+  margin-left: 3%;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+a {
+  text-decoration: none;
+  color: black;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+span {
+  border: solid grey;
   display: flex;
-  flex-direction: column;
-  align-items: center;
+  position: absolute;
+  right: 4%;
+  top: 4%;
+  text-align: center;
   justify-content: center;
-  font-size: calc(10px + 2vmin);
+  align-items: center;
+  border-radius: 50%;
+  width: 60px;
+  height: 60px;
+}
+
+/* main section */
+button {
+  cursor: pointer;
+  color: #0290ff;
+  font-size: 18px;
+  border: none;
+  background-color: #fff;
+  border-right: 1px solid grey;
+  padding: 6px 15px;
+}
+
+form {
+  display: grid;
+  width: 94%;
+  padding: 2%;
+  gap: 2%;
+  grid-template-rows: auto auto;
+  grid-template-columns: repeat(12, 1fr);
+}
+
+h2 {
+  grid-column: 1/13;
+}
+
+input,
+form > button {
+  height: 30px;
+  padding: 1%;
+  width: 100%;
+}
+
+.title {
+  grid-column: 1/7;
+}
+
+.author {
+  grid-column: 7 / span 4;
+}
+
+form > button {
+  grid-column: 11 / span 2;
+  width: 80%;
+  height: 80%;
   color: white;
+  background-color: #0290ff;
+}
+
+input {
+  border: solid #d4d4d4;
+  padding: 1%;
+}
+
+input:focus {
+  outline: none;
+}
+
+.card {
+  display: flex;
+  justify-content: space-between;
+  width: 94%;
+  padding: 2%;
+  margin: 4% auto;
+  border-radius: 5px;
+  background-color: #fff;
+}
```

Comment:
- [ ] You have a different font So stick as much as possible with the design color, font, and size.
- [ ] There is a border between the card so kindly implement and make the design the same as the design
- [ ] kindly see the spacing and make changes according to design
### your design
![hisss](https://user-images.githubusercontent.com/81727988/173073148-0303306d-315e-4b49-ac74-25f1c84dfac0.png)
### Original design
![figmmma](https://user-images.githubusercontent.com/81727988/173073220-83f6ae15-05b5-480e-a867-a297fe4d366d.png)

Reply:
@roshan-bajgain  Noted I will implement this thank you